src/resources/CaseAssistConfigs/CaseAssistConfigInterfaces.ts

```diff
@@ -54,3 +54,23 @@ export interface CaseAssistConfigModel {
     typingAidsConfiguration?: TypingAidsConfigurations;
     classificationConfigurations?: CaseClassificationConfiguration[];
 }
+
+export interface ContextField {
+    value: string;
+}
+
+export interface ContextFields {
+    [key: string]: ContextField;
+}
+
+export interface ClassifyRequestBody {
```

Comment:
This interface is similar to `DocumentsSuggestRequestBody` :arrow_down:. Could we combine them and make a reusable one?

Reply:
Both calls are accepting the same payload, so yes, we could use a single interface for both. Maybe something like `SuggestionRequestBody` would work.

However, I do prefer having distinct interfaces as both calls could accept different payloads. It is then easier to maintain compatibility.

Reply:
I plan on keeping the same interface on all suggestion calls. (that was intended)

Reply:
Given @jeremierobert-coveo comment

> I plan on keeping the same interface on all suggestion calls. (that was intended)

Alright, let's use a common interface for both calls.